src/.nuget/toolchain.ubuntu.14.04-arm.Microsoft.DotNet.ILCompiler.Development.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<package >
```

Comment:
For cross compilation, I am thinking about the following:
toolchain.ubuntu.14.04-x64.Microsoft.DotNet.ILCompiler.Arm64

This package will be restored by the CLI and based on BuildArch parameter to CLI it will pick this compiler rather than the default x64 compiler.

(Note this is still in thinking stages and needs to be made concrete with the CLI @piotrpMSFT  and @brthor. Will have a definitive answer for you this week.)

The x64 here indicates that the contents of the package run on x64 architecture. When we issue dotnet restore, from an x64 machine this package will automatically be restored and not the "ubuntu.14.04-arm".

Also, since this package contains tools for targeting ARM64, I've appended the name Arm64 to distinguish.